### PR TITLE
Added livenessprobe example with hidden parameters

### PR DESCRIPTION
**Pull request by kayrus**
Didn't find any mention about the HTTP/HTTPS  scheme or host definition. Fixed.

---

This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes.github.io/1496)

### CODE REVIEW

File: docs/user-guide/pod-states.md [hunk 1]
```diff
@@ -89,6 +89,40 @@ If a node dies or is disconnected from the rest of the cluster, some entity with
 
 ## Examples
 
+### Example manifest
```
Comment (devin-donnelly) [inline docs/user-guide/pod-states.md]: The title/header should be a little more descriptive. An example of what?

File: docs/user-guide/pod-states.md [hunk 2]
```diff
@@ -89,6 +89,40 @@ If a node dies or is disconnected from the rest of the cluster, some entity with
...
+
+Liveness probes are executed by `kubelet`, so all requests will be done within kubelet network namespace.
```
Comment (mikedanese) [inline docs/user-guide/pod-states.md]: > so all requests will be done within kubelet network namespace.

The grammar here sounds wrong.

**Comment by kayrus** [comment]
@devin-donnelly @michelleN does it look better now?

**Comment by michelleN** [comment]
I must have been tagged by accident instead of @mikedanese.
While I'm here, thanks for the clarified example @kayrus. I ran it and it worked nicely for me.

**Pull request merged**